gilded-rose-llc-frontend/README.md

```diff
@@ -0,0 +1,51 @@
+# Gilded Rose LLC Frontend
+
+This project is built using Next.js, Typescript, and Tailwind CSS.
+
+## Information on topics related to this project
+
+- [React](https://reactjs.org/)
+- [TypeScript](https://www.typescriptlang.org/)
+- [TailwindCSS](https://tailwindcss.com/docs/guides/nextjs)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)
+- [Jest](https://jestjs.io/docs/getting-started)
+
+## Getting Started
+
+Clone the repo locally
+
+```
+git clone https://github.com/jramnani/GildedRoseJavaLLC.git
+cd GildedRoseJavaLLC
+cd gilded-rose-llc-frontend
+```
+
+Install dependencies
+
+```
+yarn install
+# or
+npm install
```

Comment:
If I run `yarn install`, can I run `npm run dev` afterward? and vice versa? My inclination is that we should pick one and stick to it. Next's default package manager is `yarn` unless you create the app with the `--use-npm` flag.

Reply:
This actually came from the boilerplate README that was created when we ran `create-next-app`. I have run both yarn and npm on our current build and it works in either case. 

So I was unaware that we should stick to one. I agree that one should be shown but I wasn't sure if we could allow the developer to use their preferred tool.

Reply:
Thanks for clarifying! My concern is coming from potentially having 2 lock files: `yarn.lock` when using `yarn` and `package-json.lock` when using `npm`. [What is a lock file?](https://classic.yarnpkg.com/lang/en/docs/yarn-lock/) Each lock file will have different versions of dependencies when some devs use `yarn` and some use `npm`. My suggestion would be to stick with `yarn`.

Reply:
Now that you mention it I was getting confused why the package-json.lock would sometimes show up in my git diff with 1000+ lines changed. Its def cause sometimes I would run `npm install` or `yarn install` to try different approaches when we had an error. 

And now when I run `yarn dev` Im catching this warning on my console. 
```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```
Gotta keep a better eye on these things on my end. Thanks for mentioning this @mrfishball 

@jonmanahan Im gonna update this to only use yarn so Im going to delete the package-json.lock and reinstall yarn so it creates a new .lock file with only the yarn dependencies.

Reply:
Addressed in [this commit](https://github.com/jramnani/GildedRoseJavaLLC/pull/27/commits/25a595e62b6ef22756f2f899c9ba1c37be81903a)
- README is updated to only show run commands but also prompts the user that they can use either and just run the related npm commands
- .lock files were actually added to .gitignore since they are not needed on github. The dev cloning the repo can `yarn install` to download the dependencies when it is cloned. I removed the files from the git cache as well.

Reply:
🎉